test/npm-shrinkwrap.js

```diff
@@ -204,14 +272,21 @@ test('error on additional GIT module', fixtures(__dirname, {
         dependencies: {
             'foo': 'git://github.com:uber/foo#v1.0.0'
         },
-        'node_modules': {}
+        'node_modules': {},
+        nonono: 1
     })
 }, function (assert) {
-    npmShrinkwrap(PROJ, function (err) {
+    npmShrinkwrap(OPT, function (err) {
         assert.ok(err);
 
         assert.notEqual(err.message.indexOf(
-            'missing: foo@git://github.com:uber/foo#v1.0.0'), -1);
```

Comment:
On my machine (nvm + iojs v2.3.0 + npm 2.11.1) `err.meesage` is `foo@git://github.com/%3Auber/foo.git#v1.0.0` , which breaks this assertion.
It adds a `.git` after package name.

My guess (could be wrong) is that npm 2+ might generate a slightly different error description.